gpytorch/models/deep_gps/predictive_deep_gp.py

```diff
@@ -0,0 +1,119 @@
+import torch
+
+from gpytorch import settings
+from gpytorch.distributions import MultitaskMultivariateNormal
+from gpytorch.lazy import BlockDiagLazyTensor
+from gpytorch.likelihoods import GaussianLikelihood, MultitaskGaussianLikelihood
+
+from ..approximate_gp import ApproximateGP
+from .deep_gp import DeepGPLayer as AbstractDeepGPLayer
+
+
+class AbstractPredictiveDeepGPLayer(AbstractDeepGPLayer):
```

Comment:
Would it make sense to change the name of this class to `AbstractDSPPLayer`? cc/ @martinjankowiak

Reply:
Probably makes sense. I'll almost certainly also change the file name as well